### PR TITLE
Set up map

### PR DESCRIPTION
**Pull request by jackdavidweber**
Part of the tutorials layed out by the STEP intern program

Added markers of all places I've gone rock climbing. Markers can be hovered over for the title of the marker and clicked on for a description of the marker.
image
Screenshot 2020-06-09 at 4 21 59 PM - Display 1
image

### CODE REVIEW

File: portfolio/src/main/webapp/assets/js/map.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+var markers = [
```
Comment (augerm) [inline portfolio/src/main/webapp/assets/js/map.js]: `var` --> `const`

File: portfolio/src/main/webapp/assets/js/map.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    {lat: 40.441, lng: -75.101, title: "High Rocks", description: "Closest outdoor sport climbing from NYC. Beware the bugs."},
+]
+
+/** Creates a map that shows places I've climbed */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 39.50, lng: -98.35}, zoom: 3});
+
+  for (i=0; i<markers.length; i++){
```
Comment (augerm) [inline portfolio/src/main/webapp/assets/js/map.js]: `let i = 0` or `i` will escape into the global scope.

Comment (augerm) [reply portfolio/src/main/webapp/assets/js/map.js]: Better yet consider `for(let marker of markers)` if you don't need the index value.

**Review by augerm — APPROVED** [review]

**Review by g-harel — APPROVED** [review]

File: portfolio/src/main/webapp/assets/js/map.js (hunk 1 above)
Comment (jackdavidweber) [reply portfolio/src/main/webapp/assets/js/map.js]: got it!

**Review by Danish21 — COMMENTED** [review]
LGTM

**Pull request merged**